__tests__/app.test.js

```diff
@@ -194,117 +184,131 @@ describe("app()", () => {
           });
         });
     });
 
-    test("POST 201: /api/articles/:article_id/comments should return a 201 status code with additional request properties ignored", () => {
-      const newComment = {
-        username: "butter_bridge",
-        body: "I love this article",
-        extraProperty: "to be ignored",
-      };
+    test("PATCH 200: /api/articles/:article_id should reduce the vote count if negative number used", () => {
+      const changeVotes = { inc_vote: -10 };
       return request(app)
-        .post("/api/articles/3/comments")
-        .send(newComment)
-        .expect(201)
+        .patch("/api/articles/1")
+        .expect(200)
+        .send(changeVotes)
         .then((data) => {
           const { body } = data;
-          const comment = body.comments;
-          expect(comment).toMatchObject({
-            comment_id: 20,
-            body: "I love this article",
-            article_id: 3,
+          const article = body.updatedArticle;
+          expect(article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
             author: "butter_bridge",
-            votes: 0,
-            created_at: expect.any(String),
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 92,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
           });
-          expect(comment).not.toHaveProperty("extraProperty");
         });
     });
 
-    test("POST 400: /api/articles/:article_id/comments should return a 400 status code 'missing required field' if missing required fields", () => {
-      const newComment = {
-        body: "I love this article",
-      };
+    test("PATCH 200: /api/articles/:article_id should not reduce the vote count below 0", () => {
+      const changeVotes = { inc_vote: -200 };
       return request(app)
-        .post("/api/articles/3/comments")
-        .send(newComment)
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("missing required field");
+        .patch("/api/articles/1")
+        .expect(200)
+        .send(changeVotes)
+        .then((data) => {
+          const { body } = data;
+          const article = body.updatedArticle;
+          expect(article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
         });
     });
 
```

Comment:
A test for an incorect body would be good too for instance if the "inc_votes" key is missing or if the req body is {inc_votes: "not a number"}